### ISSUE TITLE
Ignore players that are not on the whitelist

### ISSUE THREAD

**Issue by mindw0rm** [plan-player-analytics/Plan#1810]
We have a bungee network with a whitelist (at the entry server). There are several people each day trying to connect without being on the whitelist. 
All those are shown in the playerbase, with all columns - except name, registered and country - being empty or zero: 
![grafik](https://user-images.githubusercontent.com/6404575/111925262-5fb88880-8aa8-11eb-9758-71eff4267cf4.png)

It would be nice to have those ignored in the web interface (especially player count), or even better have an option to have them deleted periodically or not stored at all.

**Comment by AuroraLS3** [comment]
After #2233 it will be possible to ignore/stop network player register if whitelist bounces

**Comment by AuroraLS3** [comment]
### TODO

- Delay player register events on proxy servers
- Run checks before player register (such as allowlist bounce) to prevent storage
